### PR TITLE
support node-pre-gyp

### PR DESCRIPTION
**Pull request by FeelyChau**

### CODE REVIEW

**Comment by FeelyChau** [comment]
Can't download the dependencies before `node-pre-gyp` fallback to build from source, so we have to implement the pre-build ourself.

File: package.json [hunk 1]
```diff
@@ -1,22 +1,24 @@
 {
   "name": "@pipcook/boa",
-  "version": "2.0.0-beta.1",
+  "version": "2.0.0",
```
Comment (yorkie) [inline package.json:3]:
```suggestion
  "version": "2.1.0",
```

**Review by WenheLI — APPROVED** [review]
LGTM

File: package.json [hunk 2]
```diff
@@ -46,20 +47,30 @@
     ]
   },
   "dependencies": {
-    "bindings": "^1.5.0",
     "debug": "^4.1.1",
     "node-addon-api": "^3.0.0"
   },
   "devDependencies": {
+    "@mapbox/node-pre-gyp": "^1.0.3",
```
Comment (yorkie) [inline package.json:54]: This needs to be in dependencies now.

**Review by yorkie — APPROVED** [review]

**Pull request merged**